github.js

```diff
@@ -178,6 +178,12 @@
          this.orgs = function(cb) {
             _request('GET', '/user/orgs', null, cb);
          };
+         
+         // List members of given organization
+         // -------
+         this.orgMembers = function (orgId, cb) {
+            _request('GET', '/organizations/' + orgId + '/members', options, cb);
```

Comment:
In passing the data to `_request` you use an `options` variable that is never declared.